packages/model-viewer/src/test/features/loading-spec.ts

```diff
@@ -135,6 +135,13 @@ suite('ModelViewerElementBase with LoadingMixin', () => {
               element.removeEventListener('load', onLoad);
             }
           });
+
+          test('getDimensions() returns correct size', () => {
```

Comment:
awesome test!

Reply:
Yeah, it's kind of lame, but at least it's something.